otter/log/cloudfeeds.py

```diff
@@ -0,0 +1,158 @@
+"""
+Ingesting events to Cloud feeds
```

Comment:
incredibly minor nitpick: I would say "Sending" or "Feeding" events to Cloud Feeds, not "Ingesting". Ingesting is what the receiver does.